frontend/src/index.css

```
@@ -211,3 +215,58 @@ body {
+
+/* Text shadow for overlay text on satellite images */
+.text-shadow-overlay {
+  text-shadow: 0 1px 3px rgb(0 0 0 / 0.8), 0 0 8px rgb(0 0 0 / 0.4);
+}
+
+/* color-mix() hover utilities — derive hover from base color */
+.hover-mix-darken {
+  transition: background-color 0.15s ease;
+}
+.hover-mix-darken:hover {
+  background-color: color-mix(in srgb, currentColor 12%, var(--tw-bg-base, transparent));
```

Stylelint (17.2.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)